src/loaders/svg/tvgSvgPath.cpp

```diff
@@ -44,12 +44,14 @@ static bool _parseNumber(char** content, float* number)
 }
 
 
-static bool _parseLong(char** content, int* number)
+static bool _parseFlag(char** content, int* number)
 {
     char* end = NULL;
-    *number = strtol(*content, &end, 10) ? 1 : 0;
-    //If the start of string is not number
-    if ((*content) == end) return false;
+    *number = strtol(*content, &end, 10);
+    //If the start of string is not number or a number was a float
+    if ((*content) == end || *end == '.') return false;
+    //If a flag has a different value than 0 or 1
+    if (*number != 0 && *number != 1) return false;
```

Comment:
If it is not 0 or 1, skipComma does not work and I think it may affect the content.

Reply:
If the value is not 0 or 1, _parseFlag will return false and in a consequence _nextCommand will return NULL, which will break the while loop in svgPathToTvgPath and any further commands from the svg path (content) will not be analysed. And this is the expected behavior according to the spec.